pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.cloudsoft.brooklyn</groupId>
+    <artifactId>web-plugin</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <modules>
+        <module>web-plugin-endpoint</module>
+        <module>web-plugin-feature</module>
+    </modules>
+    <packaging>pom</packaging>
+
+    <properties>
+        <brooklyn.version>0.11.0</brooklyn.version>  <!-- BROOKLYN_VERSION -->
```

Comment:
When I had this as the parent and used `<packaging>bundle</packaging>` it produced the error `The packaging for this project did not assign a file to the build artifact`.